topsort-endpoints.yml

```
@@ -210,25 +210,25 @@ components:
           example: v_89kjm
 
     Session:
-      description: The Session object allows correlating user activity during a session whether or not they are actually logged in. For all calls to Topsort, include as many as are known.
+      description: The Session object allows correlating user activity during a session whether or not they are actually logged in.
       required:
         - sessionId
       type: object
       properties:
         sessionId:
+          description: >
+            Long-lived token identifying the customer interacting with the marketplace.
+
+            If your users are always logged in you could use a hash of your customer ID.
```

What do you think about this @fcifuentesq ?
```suggestion
            If your users are always logged in you may use a hash of your customer ID, in which case the optional `consumerId` would not be needed.
```

---

I like it!